add-on/_locales/en/messages.json

```
@@ -472,16 +480,20 @@
     "description": "Status label on the share files page (quickUpload_state_buffering)"
   },
   "quickUpload_options_show": {
-    "message": "upload options",
+    "message": "import options",
     "description": "Button on the share files page (quickUpload_options_show)"
   },
+  "quickUpload_options_uploadDir": {
+    "message": "MFS directory to store imported files",
+    "description": "Textbox label on the share files page (quickUpload_options_uploadDir)"
+  },
   "quickUpload_options_wrapWithDirectory": {
     "message": "Wrap single files in a directory to preserve their filenames.",
     "description": "Checkbox label on the share files page (quickUpload_options_wrapWithDirectory)"
   },
-  "quickUpload_options_pinUpload": {
-    "message": "Pin files so they are retained when performing garbage collection on your IPFS repo.",
-    "description": "Checkbox label on the share files page (quickUpload_options_pinUpload)"
+  "quickUpload_options_openViaWebUI": {
+    "message": "Open imported files in Web UI",
```

"imported files" is already in `quickUpload_options_uploadDir`, so we can make this one shorter or need to rephrase it
```suggestion
    "message": "Open in Web UI",
```